lib/cylc/gui/dbchooser.py

```diff
@@ -105,8 +105,11 @@ def update(self):
 
         # Scan for running suites
         choices = []
-        for host, port, suite_identity in scan_many(
-                get_scan_items_from_fs(), timeout=self.timeout):
+        scan_results = scan_many(
+            get_scan_items_from_fs(), timeout=self.timeout)
+        if scan_results is None:
+            return
```

Comment:
It's probably better to ensure that `scan_many()` returns an empty list as intended if nothing is found, else we may get the same problem with `cylc scan`.